README.md

```diff
@@ -26,10 +26,11 @@ jobs:
 
 | name | value | default | description |
 | ---- | ----- | ------- | ----------- |
-| github_token | string | | Token for the repo. Can be passed in using {{ secrets.GITHUB_TOKEN }}'. |
+| github_token | string | | Token for the repo. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`. |
```

Comment:
Not sure about this, and this is a bug from the past, but shouldn't this be `${{ <expression> }}`?